src/img/components/icons/icon-system/spinner.svg

```diff
@@ -0,0 +1,33 @@
+<svg class="lds-spinner" width="200px"  height="200px"  xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid" style="background: none;"><g transform="rotate(0 50 50)">
+  <rect x="43" y="2" rx="129" ry="6" width="14" height="28" fill="#1B2029">
+    <animate attributeName="opacity" values="1;0" times="0;1" dur="0.7s" begin="-0.6124999999999999s" repeatCount="indefinite"></animate>
```

Comment:
@Poltergeist the css animations did not work since we are rendering the svg as shadow DOM and even trying to add scope css for some reason did not get parsed/used.

If you have a working suggestion following the same design required by @leemunroe please suggest

Reply:
@weblancaster if you haven't tested this animation across the browsers we support, we certainly should!

Reply:
the animation works fine in all browsers but IE edge but I wouldn't block the PR because of that.

Reply:
@weblancaster let's move forward with the PR. I've created an issue for us to revisit the spinner https://jira.mesosphere.com/browse/DCOS-19469

Reply:
ok thanks!